src/main/java/com/andersen/java_intensive_13/hotel_app/model/Apartment.java

```diff
@@ -0,0 +1,30 @@
+package com.andersen.java_intensive_13.hotel_app.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.persistence.*;
+import lombok.*;
+
+@Getter
+@Setter
+@ToString
```

Comment:
here could be StackOverflowException due private User user;
exclude it from ToString